src/c++/perf_analyzer/client_backend/client_backend.h

```diff
@@ -175,15 +196,23 @@ class ClientBackendFactory {
   /// \param http_headers Map of HTTP headers. The map key/value
   /// indicates the header name/value. The headers will be included
   /// with all the requests made to server using this client.
+  /// \param server_library_path Only for C api backend. Lbrary path to
```

Comment:
It probably makes more sense to say "triton_server_path" and have it be the path to the top-level Triton directory (which is typically /opt/tritonserver, because you need more than just libtritonserver.so for triton to work. You can then derive the path to libtritonserver.so from thiat.